### ISSUE TITLE
Auth Backends Not Compatible with Django 2.1

### ISSUE THREAD

**Issue by paltman** [pinax/django-user-accounts#292]
Not sure what the right solution is in terms of supporting <2.1 and 2.1 but to get my current project to work with 2.1, I did the following:

```python
# backends.py
from account.auth_backends import UsernameAuthenticationBackend as UAB


class UsernameAuthenticationBackend(UAB):

    def authenticate(self, request, **credentials):
        return super().authenticate(**credentials)
```

Then I installed that auth backend in settings.py instead of the DUA one.

For reference, it was [5e31be1b96f60232e1e04140c5f0e33d8c2319f1](https://github.com/django/django/commit/5e31be1b96f60232e1e04140c5f0e33d8c2319f1) that started requiring `request` on `authenticate` methods.

**Comment by tubaman** [comment]
How about this? https://github.com/pinax/django-user-accounts/pull/293

**Comment by tubaman** [comment]
@etcware, this is probably the wrong forum for general pinax support.  You might want to try the [slack channel](https://pinax.slack.com/messages/C11N6FU56/)

**Comment by justb4** [comment]
I can confirm @paltman 's suggested fix works with DJ 2.1.2. It basically caters for the changed signature in `authenticate()` as treated in PR #293 . Tried out Pinax `account`, got it working with these steps: 

1) Create a file `backends.py` within the app-dir (mine is named `accountdemo` after installing with `pinax start account accountdemo`) where `settings.py` etc is with contents:
```
# backends.py
from account.auth_backends import UsernameAuthenticationBackend as UAB


class UsernameAuthenticationBackend(UAB):

    def authenticate(self, request, **credentials):
        return super().authenticate(**credentials)
```

2) Install by making the last lines of `settings.py` as follows:


```
AUTHENTICATION_BACKENDS = [
    "accountdemo.backends.UsernameAuthenticationBackend",
    # "account.auth_backends.UsernameAuthenticationBackend",
]
```
Tip: when trying signup, email confirmation may fail, but in the logs you may see a URL like:
http://localhost:8000/account/confirm_email/c5f1b8c..longhexstring/. Paste that in the browser and the signup process succeeds!
